ecs-cli/modules/compose/ecs/utils/convert_task_definition_test.go

```diff
@@ -387,18 +406,23 @@ func TestConvertToMountPoints(t *testing.T) {
 	}
 }
 
-func verifyMountPoint(t *testing.T, output *ecs.MountPoint, volumes map[string]string,
-	hostPath, containerPath string, readonly bool) {
-
+func verifyMountPoint(t *testing.T, output *ecs.MountPoint, volumes *volumes,
+	hostPath, containerPath string, readonly bool, emptyHostCtr *int) {
+	sourceVolume := ""
 	if containerPath != *output.ContainerPath {
 		t.Errorf("Expected containerPath [%s] But was [%s]", containerPath, *output.ContainerPath)
 	}
-	sourceVolume := volumes[hostPath]
+	if hostPath != "" {
+		sourceVolume = volumes.volumeWithHost[hostPath]
+	} else {
+		sourceVolume = volumes.volumeEmptyHost[*emptyHostCtr]
```

Comment:
I think that hard-coding this index might be better than incrementing using a pointer. We already hardcode the index for the `mountPointsOut` slice when this is invoked. What do you think about not incrementing this variable, but passing in a hard-coded value from where it's invoked? So it'd be `verifyMountPoint(t *testing.T, output *ecs.MountPoint, volumes *volumes, hostPath, containerPath string, readonly bool, emptyHostIndex int)`. And then you'll invoke it with the appropriate index from `TestConvertToMountPoints()`. Does that make sense?